src/fprime_gds/executables/cli.py

```diff
@@ -651,6 +651,14 @@ def get_arguments(self) -> Dict[Tuple[str, ...], Dict[str, Any]]:
                     "type": str,
                     "help": "Path to packet specification.",
                 },
+                ("--packet-set-name",): {
+                    "dest": "packet_set_name",
+                    "action": "store",
+                    "default": None,
+                    "required": False,
+                    "type": str,
+                    "help": "Packet set name",
```

Comment:
Can we add a few words explaining what this is and when it's needed? Shorter the better but this may confuse users who aren't familiar with it.

Reply:
Sure, I'll update this.